.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.145.1/containers/javascript-node/.devcontainer/base.Dockerfile
```

Comment:
For use by GH codespaces. Helps set up the environment a user gets by default when they open our repo in a codespaces.